src/components/molecules/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -45,30 +50,30 @@ const MenuItemLink = ({ text, link }: { text: string; link: string }) => {
     );
 };
 
-const MobileItemLink = ({ text, href }: { text: string; href: string }) => {
+const MobileItemLink = ({
+    text,
+    href,
+    target,
+}: {
+    text: string;
+    href: string;
+    target?: string;
+}) => {
+    const router = useRouter();
+    const isActive = router.pathname === href;
     return (
         <Menu.Item>
-            {({ active }) => (
-                <a
-                    className={clsx(
-                        'flex h-16 w-full flex-row items-center justify-start gap-3 whitespace-nowrap p-4 text-center',
-                        {
-                            'rounded-2xl bg-[#233447] text-neutral-8': active,
-                        }
-                    )}
-                    href={href}
-                    target='_blank'
-                    rel='noreferrer'
-                >
-                    <p>{text}</p>
-                    <ArrowUpIcon
```

Comment:
Also removed this icon as external links don't have the arrow in the new design:

https://www.figma.com/design/YIBsk1ihFbPlDb8XldGj1T/SF-Design-System?node-id=691-11952&m=dev